packages/web/src/components/molecules/Console/ConsoleLines.tsx

```diff
@@ -9,11 +9,23 @@ export const ConsoleLines: FC<{
   start: number;
   maxDigits: number;
   LineComponent: ConsoleProps['LineComponent'];
-}> = memo(({lines, start, maxDigits, LineComponent = ConsoleLine}) => (
+  onSelectLine?: (line: number) => void;
+  selectedLine?: number;
```

Comment:
I'm not sure if there should be `selectedLine` at all, but if so, it should be `selected: boolean` instead

Reply:
We can't really have `selected`, since in this component is about multiple lines, and we would need to check the selected line with the rendered one 🤔

Reply:
Update: after latest commit this prop has been removed